voctocore/default-config.ini

```diff
@@ -43,10 +43,12 @@ mix_out=10000
 ;default-b=cam2
 
 [side-by-side-preview]
-;asize=1024x576
-;apos=12/12
-;bsize=320x180
-;bpos=948/528
+asize=1536x864
+acrop=0/0/0/0
+apos=36/160
+bsize=256x432
+bcrop=0/640/0/640
+bpos=1620/506
```

Comment:
I think we want these to be commented in the default config.
Also this needs a comment providing documentation. Without reading the code it's far from obvious that this is
1. the amount being cropped, and not e.g. a rect
2. ordered top/left/bottom/right